impl/client/src/main/javascript/web/pentaho/visual/models/Boxplot.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2023 Hitachi Vantara. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/module!_",
+  "./CategoricalContinuousAbstract",
+  "pentaho/i18n!./i18n/model",
+  "./types/LabelsOption"
+], function(module, BaseModel, bundle, LabelsOption) {
+
+  "use strict";
+
+  return BaseModel.extend({
+    $type: {
+      id: module.id,
+      category: "misc2",
+
+      props: [
+        // VISUAL_ROLES
+        {
+          name: "rows",
+          modes: {dataType: "list"},
+          fields: {isRequired: true}
+        },
+        {
+          name: "lowerQuartile",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 11
+        },
+        {
+          name: "upperQuartile",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 12
+        },
+        {
+          name: "median",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 13
+        },
+        {
+          name: "minimum",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 14
+        },
+        {
+          name: "maximum",
+          base: "pentaho/visual/role/Property",
+          modes: {dataType: "number"},
+          ordinal: 15
+        },
+        {
+          name: "measures",
+          isBrowsable: false
+        },
```

Comment:
Perhaps `measures` could be used instead of the new `median` visual role? Changing the label and the relative order, `ordinal`, of the other new roles, so that these are placed around it.
It's not great to have visual roles which are never used. Don't think there is the precedent. For the multi-chart VR, for example, there is even a mixin, so that it can be reused and mixed on the cases that have it.

If really need to go with this solution, the `isApplicable: false` property should be used instead

Reply:
Did the required changes by using 'measures' and mapping it to 'median'